features/performance/manifests/01-tuned-rt.yaml

```diff
@@ -23,10 +23,15 @@ spec:
       vm.stat_interval = 10
       kernel.timer_migration = 0
 
+      [sysfs]
+      /sys/bus/workqueue/devices/writeback/cpumask = ${not_isolated_cpumask}
+      #/sys/devices/virtual/workqueue/cpumask = ${not_isolated_cpumask}
```

Comment:
any reason to keep commented code around?

Reply:
@fromanirh  maybe as a reference of what was done for RHEL but now its done with this patch so ill remove it